cugl/lib/2d/CUNode.cpp

```diff
@@ -221,7 +221,7 @@ bool Node::initWithData(const SceneLoader* loader, const std::shared_ptr<JsonVal
     if (data->has("angle")) {
         transform = true;
         _angle = data->getFloat("angle",0.0f);
-        _angle = M_PI/180.0f;
+        _angle *= M_PI/180.0f;
```

Comment:
You better believe I'm mentioning this in the postmortem today

Reply:
cugl? more like bugl amirite